stress-ng.c

```diff
@@ -652,6 +652,10 @@ static const struct option long_options[] = {
 	{ "sched-prio",	1,	0,	OPT_sched_prio },
 	{ "schedpolicy",1,	0,	OPT_schedpolicy },
 	{ "schedpolicy-ops",1,	0,	OPT_schedpolicy_ops },
+	{ "sched-period",1,	0,	OPT_sched_period },
+	{ "sched-runtime",1,	0,	OPT_sched_runtime },
+	{ "sched-deadline",1,	0,	OPT_sched_deadline },
+	{ "schedpolicy",1,	0,	OPT_schedpolicy },
 	{ "sctp",	1,	0,	OPT_sctp },
```

Comment:
These new options need to be in the manual with suitable description to what they do and what the defaults are.

Reply:
Got it. I'll make the manual in v2.